GOVERNANCE.md

```diff
@@ -1,45 +1,107 @@
+
+For the first formation of the committee, our maintainers nominated a group of people who had great impact or influence on the project, including some members from PingCAP who founded TiKV. New PMC members are selected based on the rules described above. The term is one year.
 
-PMC: The Project Management Committee; functions as the core management team that oversees the TiDB community. The PMC decides the roadmap of the project and makes major decisions related to the community.
+The PMC is responsible for maintaining the [Guiding Principles](/guiding-principles.md) and the code of conduct. It is also responsible for ensuring that those rules and principles are followed.
 
-Developer Group:
+For a list of the current PMC members, see the [PMC Members](/committee/README.md#members).
 
-- Maintainer: Mainatiners are the planners and designers of the project, with the authority to merge branches into the master. 
-- Committer:  Recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
-- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
-- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
-- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+### Developer Group
 
-Organizer Committee:  The Organizer Committee consists of organizers in charge of event or activity operations; in charge of the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions
+As the cornerstone of community development, the TiKV Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiKV. For more details, refer to [Community Membership](/community-membership.md).
 
-## Developer Group Structure
+The Developer Group operates TiKV projects in two groups: Special Interest Group (SIG) and Working Group (WG). The diagram below illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion path.
 
 ![developer group](/media/governance/developer_group.png)
 
-Community groups include Special Interest Group (SIG) and Working Group (WG). The second part of the organizational structure illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion machanism.
+### Maintainers
+
```

Comment:
I think Maintainers are still the core group here, as TiKV is pretty hardcore in the technical sense. This is so that Maintainers can be focus on the tech aspect of the work and let PMC handle other governance work.